index.html

```diff
@@ -35,9 +35,9 @@
                      alt="Banner"/>
             </picture>
             <div class="text">
-                <p class="title">HALLO UND WILLKOMMEN BEI "EIN TIERISCHER SPAß"</p>
-                <p class="body">Auf dieser Seite möchten wir dir ein paar <strong>tolle Tiere vorstellen</strong>. Dazu musst du einige <strong>Rätsel</strong> lösen und spannende <strong>Spiele</strong> absolvieren. Es gibt <strong>drei Schwierigkeitsgrade</strong> die jeweils <strong>9 Spiele</strong> beinhalten. Nach jedem <strong>Spiel</strong> erhält du einen Link, der dich zu einem <strong>geheimen Puzzleteil</strong> führt. Wenn du alle <strong>Spiele</strong> erfolgreich absolviert hast, wartet auf dich ein richtiger Knobelspaß. Lass dich überraschen. Wir wünschen dir <strong>viel Spaß</strong> beim Lernen!</p>
-                <p class="greetings">Dein*e Nora, Bryan & Lizzy</p>
+                <p class="title">HALLO UND WILLKOMMEN BEI: "EIN TIERISCHER SPAß!"</p>
+                <p class="body">Auf dieser Seite möchten wir dir ein paar <strong>tolle Tiere</strong> vorstellen. Dazu musst du einige <strong>Rätsel</strong> lösen und spannende <strong>Spiele</strong> spielen. Es gibt <strong>drei Schwierigkeitsgrade<strong></strong>, in denen jeweils <strong>9 Spiele</strong> zu finden sind. Nach jedem Spiel erhältst du einen Link, der dich zu einem <strong>geheimen Puzzleteil</strong> führt. Wenn du alle Spiele erfolgreich gemeistert hast, wartet auf dich ein richtiger Knobelspaß. Lass dich überraschen! Wir wünschen dir <strong>viel Spaß</strong> beim Entdecken und Lernen!</p>
```

Comment:
`<strong> blabla <strong></strong> blabla` ist nicht gut, sollte korrigiert werden

Reply:
Fixed it